pkg/config/config.go

```diff
@@ -54,5 +55,9 @@ func validateConfig(c Config) []error {
 	if c.GithubHubAppConfig.OAuth.ClientID == "" || c.GithubHubAppConfig.OAuth.ClientSecret == "" {
 		errs = append(errs, errors.ConfigNotValidError("you must provide client_id / client_secret in github_app_config.oauth field"))
 	}
+
+	if c.SubFolderParallelism == 0 {
+		c.SubFolderParallelism = 1
+	}
```

Comment:
weird stuff

Reply:
fixed by setting this field as mandatory